osu.Desktop/DiscordRichPresence.cs

```diff
@@ -141,7 +192,7 @@ private string getDetails(UserActivity activity)
             switch (activity)
             {
                 case UserActivity.InGame game:
-                    return game.BeatmapInfo.ToString();
+                    return $"[{Math.Round(game.BeatmapInfo.StarRating, 2)}*] {game.BeatmapInfo}";
```

Comment:
Please do not add unrelated changes to diffs like this. This change has nothing to do with the PR's title and wasn't even properly discussed as an addition before.

Reply:
Oh ok, do you think I should update the PR's name to include this change as well or do you think that it's not a relevant change at all ?

Reply:
Would revert for now and re-raise in a separate discussion thread.